src/libsync/configfile.cpp

```diff
@@ -49,56 +49,60 @@ namespace chrono = std::chrono;
+const QString forceSyncIntervalC() { return QStringLiteral("forceSyncInterval"); }
+const QString fullLocalDiscoveryIntervalC() { return QStringLiteral("fullLocalDiscoveryInterval"); }
+const QString notificationRefreshIntervalC() { return QStringLiteral("notificationRefreshInterval"); }
+const QString monoIconsC() { return QStringLiteral("monoIcons"); }
+const QString promptDeleteC() { return QStringLiteral("promptDeleteAllFiles"); }
+const QString crashReporterC() { return QStringLiteral("crashReporter"); }
+const QString optionalDesktopNoficationsC() { return QStringLiteral("optionalDesktopNotifications"); }
+const QString showInExplorerNavigationPaneC() { return QStringLiteral("showInExplorerNavigationPane"); }
+const QString skipUpdateCheckC() { return QStringLiteral("skipUpdateCheck"); }
+const QString updateCheckIntervalC() { return QStringLiteral("updateCheckInterval"); }
+const QString updateChannelC() { return QStringLiteral("updateChannel"); }
+const QString geometryC() { return QStringLiteral("geometry"); }
+const QString timeoutC() { return QStringLiteral("timeout"); }
+const QString chunkSizeC() { return QStringLiteral("chunkSize"); }
+const QString minChunkSizeC() { return QStringLiteral("minChunkSize"); }
+const QString maxChunkSizeC() { return QStringLiteral("maxChunkSize"); }
+const QString targetChunkUploadDurationC() { return QStringLiteral("targetChunkUploadDuration"); }
+const QString automaticLogDirC() { return QStringLiteral("logToTemporaryLogDir"); }
+const QString deleteOldLogsAfterHoursC() { return QStringLiteral("temporaryLogDirDeleteOldLogsAfterHours"); }
+const QString showExperimentalOptionsC() { return QStringLiteral("showExperimentalOptions"); }
+const QString clientVersionC() { return QStringLiteral("clientVersion"); }
+
+const QString proxyHostC() { return QStringLiteral("Proxy/host"); }
+const QString proxyTypeC() { return QStringLiteral("Proxy/type"); }
+const QString proxyPortC() { return QStringLiteral("Proxy/port"); }
+const QString proxyUserC() { return QStringLiteral("Proxy/user"); }
+const QString proxyPassC() { return QStringLiteral("Proxy/pass"); }
+const QString proxyNeedsAuthC() { return QStringLiteral("Proxy/needsAuth"); }
+
+const QString useUploadLimitC() { return QStringLiteral("BWLimit/useUploadLimit"); }
+const QString useDownloadLimitC() { return QStringLiteral("BWLimit/useDownloadLimit"); }
+const QString uploadLimitC() { return QStringLiteral("BWLimit/uploadLimit"); }
+const QString downloadLimitC() { return QStringLiteral("BWLimit/downloadLimit"); }
+
+const QString newBigFolderSizeLimitC() { return QStringLiteral("newBigFolderSizeLimit"); }
+const QString useNewBigFolderSizeLimitC() { return QStringLiteral("useNewBigFolderSizeLimit"); }
+const QString confirmExternalStorageC() { return QStringLiteral("confirmExternalStorage"); }
+const QString moveToTrashC() { return QStringLiteral("moveToTrash"); }
```

Comment:
well it could also mean constant, so other projects have the convention to start constants uppercase, so I thought we could keep it.
Also it would require me to fiddle with regex again and... :D

Reply:
okay ... whatever